Readme.Md

```diff
@@ -0,0 +1,187 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Live Demo ](#live-Demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Mtv-Shows] <a name="about-project"></a>
+
+**[Mtv-Shows]** is a website that allows you to see the most popular shows in different countries and let you comment on them and also reserve them
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+     <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">CSS</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/Javascript">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[User can Comment and all the comments can be seen realtime]**
+- **[User the Reserve and all the reservation can be seen ]**
+- **[Using the Api user can add likes to their favourite show]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Live Demo <a name="getting-started"></a>
+
+## Link: https://www.loom.com/share/2ae8bf18c82b4c3988c2253a90fc4c0d?sid=0885c5d9-6594-4073-8adc-46521cc5379f
+
```

Comment:
- [ ] You did a great job recording a video, unfortunately, the video did not meet all the requirements such as showing a piece of code you are proud of or everyone appearing in the video (you are a team of three members), etc.


![videojs](https://github.com/brighton0102/JavascriptCapstone/assets/53124548/2f8eaa2e-18a5-4b5c-82ef-af251092cf3e)